.github/workflows/connect-dev-release.yml

```diff
@@ -0,0 +1,127 @@
+      - "yarn.lock"
+
+env:
+  DEV_SERVER_HOSTNAME: "dev.suite.sldev.cz"
+
+jobs:
+  build-connect-explorer:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          lfs: true
+
+      - name: Configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::538326561891:role/gh_actions_trezor_suite_dev_deploy
+          aws-region: eu-central-1
+
+      - name: Extract branch name
+        run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
+        id: extract_branch
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: Build connect-web
+        run: |
+          git submodule update --init --recursive
+          yarn install --immutable
+          yarn build:libs
+          yarn workspace @trezor/connect-web build
+          ./packages/connect-web/scripts/check-inline-build-size.sh
+          yarn workspace @trezor/connect-iframe build
+          yarn workspace @trezor/connect-popup build
+          yarn workspace @trezor/connect-webextension build
+
+      - name: Build connect-explorer
+        env:
+          ASSET_PREFIX: /connect/${{ steps.extract_branch.outputs.branch }}
+          # TODO: maybe we don't want to build it for production for every branch
+          NODE_ENV: "production"
+          name: $CI_COMMIT_REF_NAME
+          url: https://$DEV_SERVER_HOSTNAME/connect/$CI_COMMIT_REF_NAME
+        run: |
+          yarn workspace @trezor/connect-explorer build
+
+      - name: Build connect-examples webextension
+        run: |
+          node ./packages/connect-examples/update-webextensions.js --trezor-connect-src "https://$DEV_SERVER_HOSTNAME/connect/${CI_COMMIT_REF_NAME}/"
+          node ./packages/connect-examples/update-webextensions.js --trezor-connect-src "https://$DEV_SERVER_HOSTNAME/connect/${CI_COMMIT_REF_NAME}/" --build-folder build-legacy --npm-src "https://connect.trezor.io/9/trezor-connect.js"
+
+      - name: Build connect-explorer-webextension
+        run: |
+          yarn workspace @trezor/connect-explorer build:webextension
+
+      # this step should upload build result to s3 bucket DEV_SERVER_HOSTNAME using awscli
+      - name: Upload connect-explorer to dev.suite.sldev.cz
+        env:
+          DEPLOY_PATH: s3://dev.suite.sldev.cz/connect/${{ steps.extract_branch.outputs.branch }}
+        run: |
+          mkdir -p tmp_build_directory
+          cp -R ./packages/connect-iframe/build/* tmp_build_directory/
+          cp -R ./packages/connect-popup/build/* tmp_build_directory/
+          cp -R ./packages/connect-web/build/* tmp_build_directory/
+          cp -R ./packages/connect-explorer/build/* tmp_build_directory/
+          aws s3 sync --delete tmp_build_directory/ ${DEPLOY_PATH}
+
+      - name: Zip and Upload Artifacts
```

Comment:
you shouldn't need to do this. It will zip up artifacts for you in the `actions/upload-artifact` action and will put it there under one zip archive with the name you set in 
```yaml
        with:
          name: test-artifacts
```

Reply:
> you shouldn't need to do this. It will zip up artifacts for you in the `actions/upload-artifact` action and will put it there under one zip archive with the name you set in
> 
> ```yaml
>         with:
>           name: test-artifacts
> ```

Yes it works as you said! Thanks!